.github/workflows/system_tests.yml

```diff
@@ -78,10 +78,10 @@ jobs:
             echo "miner docker image NOT found for [${{ env.TAG }}-${{ env.SHORT_SHA }}]"
             git fetch origin pull/${{ steps.findPr.outputs.pr }}/merge:merge && git checkout merge
             MERGE_SHORT_SHA=$(git rev-parse HEAD | head -c 8)
-            echo "Checking miner image [refs-pull-${{ steps.findPr.outputs.pr }}-merge-$MERGE_SHORT_SHA] is present..."
-            docker pull 0chaindev/miner:refs-pull-${{ steps.findPr.outputs.pr }}-merge-$MERGE_SHORT_SHA || { echo "Docker image for miner could not be found for either branch commit sha [${{ env.SHORT_SHA }}] or PR merge commit sha [$MERGE_SHORT_SHA]. Please publish image manually at https://github.com/0chain/0chain/actions/workflows/build-&-publish-docker-image.yml and try again." && exit 1; }
-            echo "miner docker image found for  [refs-pull-${{ steps.findPr.outputs.pr }}-merge-$MERGE_SHORT_SHA]"
-            echo "MINER_IMAGE=refs-pull-${{ steps.findPr.outputs.pr }}-merge-$MERGE_SHORT_SHA" >> $GITHUB_ENV
+            echo "Checking miner image [merge-$MERGE_SHORT_SHA] is present..."
```

Comment:
the old naming strategy was better because it included the PR number. Now every PR is going to be pushing to "merge" meaning merge latest is meaningless

Reply:
we need to fix this so will merge now

Reply:
hmm. `mrege-id` should be better